hack/setup-kind-in-ci.sh

```diff
@@ -245,6 +245,7 @@ setup_kind_singlecluster() {
     --set health_config.rate[0].tolerance[0].degraded=2 \
     --set health_config.rate[0].tolerance[0].failure=100 \
     --set kiali_internal.cache_expiration.ambient_check="10s" \
+    --set kiali_internal.cache_expiration.istio_status="10s" \
```

Comment:
If you think these values are too high, still, go ahead and change to 5s.  But there is a problem, you need to add the new setting in this file in two places.